### PR TITLE
 PR for adding dropdowns, sharing button & 'add articles' button 

### PR DESCRIPTION
**Pull request by MeghaSharma21**
@eggpi please review.
PR contains the code for the following:
1. Replaced navigation buttons with drop-downs for UX consistency.
2. Added button for sharing worklists on tasks page
3. Added button for adding articles to the worklist on tasks page
4. Resolved bug: Login button showed up despite being logged-in on 'my worklists' page

### CODE REVIEW

File: static/js/show-task.js [hunk 1]
```diff
@@ -59,3 +59,39 @@ function worker() {
...
+                refresh();
+            }
+        },
+        error: function () {
+            $("#alert-message-holder").html(
+                "<div class='alert alert-danger' role='alert'><a href='#' class='close' " +
+                "data-dismiss='alert'>&times;</a> " +
+                "<strong>Oh snap!</strong>Something went wrong while" +
+                " saving updated information! Please report Megha " +
+                "at meghasharma4910@gmail.com</div>"
```
Comment (eggpi) [inline static/js/show-task.js]: No need to do it now, but probably better to just link to this repository somewhere on the tool and let people file issues. That way you don't have to use your email as issue tracker :)

File: templates/show-task.html [hunk 2]
```diff
@@ -36,13 +39,17 @@
 
 {% block content %}
 	<div class="container">
+        <div id="alert-message-holder"></div>
 		<div class="row">
 			<div class="col-md-12 create-worklist-container">
-				<h2 class="top-heading">Worklist Name: {{worklist_name}}</h2>
+                <h2 class="top-heading">Worklist Name: {{worklist_name}}</h2>
+                <a href="#" data-toggle="popover" data-content="Share Worklist using this link: https://tools.wmflabs.org/worklist-tool/show-tasks?worklist_name={{ worklist_name }}&worklist_created_by={{ worklist_created_by }}"><i class="fa fa-share-alt" class="share-icon-size"></i></a>
```
Comment (eggpi) [inline templates/show-task.html]: I think we were discussing a shorter URL like on GitHub repositories, like:

https://tools.wmflabs.org/worklist-tool/worklist/{{ user }}/{{ worklist }}

How about that?

File: templates/show-worklist.html [hunk 3]
```diff
@@ -14,6 +14,10 @@
     var awesomplete_by_username;
 
     $(document).ready(function(){
+
+    	// making the nav active
```
Comment (eggpi) [inline templates/show-worklist.html]: This comment doesn't add much to what I can already read on the line below. Remove?

File: static/js/show-task.js (hunk 1 above)
Comment (MeghaSharma21) [reply static/js/show-task.js]: I've placed a button on the header. Does it look fine?

File: templates/show-task.html (hunk 2 above)
Comment (MeghaSharma21) [reply templates/show-task.html]: Done.

File: templates/show-worklist.html (hunk 3 above)
Comment (MeghaSharma21) [reply templates/show-worklist.html]: Done.

File: static/js/show-task.js (hunk 1 above)
Comment (eggpi) [reply static/js/show-task.js]: Sure!

File: templates/show-task.html [hunk 4]
```diff
@@ -36,13 +39,17 @@
 
 {% block content %}
 	<div class="container">
+        <div id="alert-message-holder"></div>
 		<div class="row">
 			<div class="col-md-12 create-worklist-container">
-				<h2 class="top-heading">Worklist Name: {{worklist_name}}</h2>
+                <h2 class="top-heading">Worklist Name: {{worklist_name}}</h2>
+                <a href="#" data-toggle="popover" data-content="Share Worklist using this link: https://tools.wmflabs.org/worklist-tool/show-tasks/{{ worklist_created_by }}/{{ worklist_name }}"><i class="fa fa-share-alt" class="share-icon-size"></i></a>
```
Comment (eggpi) [inline templates/show-task.html]: Can the link be wrapped in `<a>`?

Comment (eggpi) [inline templates/show-task.html]: Personal preference, but I think I'd make the share button larger (people tend to not find small buttons) and possibly align it to the right, above the Search button.

**Comment by MeghaSharma21** [comment]
If this looks fine, I'll merge and send another PR.

File: templates/show-task.html (hunk 4 above)
Comment (MeghaSharma21) [reply templates/show-task.html]: Done

Comment (MeghaSharma21) [reply templates/show-task.html]: Done!

**Comment by eggpi** [comment]
Sure, go for it!

**Pull request merged**